client/devdocs/design/index.jsx

```diff
@@ -83,44 +97,44 @@ export default React.createClass( {
 					</SearchCard>
 				}
 				<Collection component={ this.props.component } filter={ this.state.filter }>
-					<Accordions />
```

Comment:
I think this what you're trying to do with [```React.cloneElement```](https://facebook.github.io/react/docs/top-level-api.html#react.cloneelement)

```js
const newChildren = React.Children.map( this.props.children, function( child ) {
  return React.cloneElement( child, { componentUsageStats: componentsUsageStats[ ... ] } );
});
```
However this creates a copy of each element in memory, perhaps not so good. Can you map an array of Components in this render function? [html-tags-vs.-react-components](https://facebook.github.io/react/docs/jsx-in-depth.html#html-tags-vs.-react-components)

```js
render () {
  const children = [
    Accordions,
    BulkSelect,
    ButtonGroups,
    Buttons,
    Cards,
    ...
  ];
 return (
 	<Collection>
  	{
    	children.map( Item => {
      	return <Item componentUsageStats={ camelCase( Item.displayName ) }
      } );
    }
  </Collection>
 );
}
<Collection